src/components/Map.js

```diff
@@ -113,7 +113,7 @@ export default function Map() {
     fillOpacity: 1,
   };
 
-  const heatmap = ["#FFED39", "#EB4F12", "#D84091", "#8E47F3", "#0852C1"];
+  const heatmap = ["#0852C1", "#8E47F3", "#D84091", "#EB4F12", "#FFED39"];
```

Comment:
Just wondering why #0852C1 is used here instead of #0D277C?

Reply:
Whoops! I switched to #0852C1 because #0D277C was rendering pretty dark on the map. Just updated the key so it's the same color. Thanks for the catch!!